frontend/src/types/api/queryBuilder/getAttributeKeys.ts

```diff
@@ -3,9 +3,10 @@ import { DataSource } from 'types/common/queryBuilder';
 import { BaseAutocompleteData } from './queryAutocompleteResponse';
 
 export interface IGetAttributeKeysPayload {
-	aggregateOperator: string;
+	aggregateOperator?: string;
```

Comment:
please make a new API and types to use this. adding optional checks on these makes the whole structure loosely typed

Reply:
Done @vikrantgupta25